src/main/java/edu/iis/mto/time/Order.java

```diff
@@ -24,21 +24,29 @@ public void addItem(OrderItem item) {
 
 	}
 
-	public void submit() {
+	public void submit(DateTime dateTimeTest) {
```

Comment:
to nie jest dobre rozwiązanie kiedy parametr metody jest tak na prawdę wskaźnikiem sposobu uruchomienia - gdyby nie było testów to byłby zbędny